### ISSUE TITLE
Links in readme not working

### ISSUE THREAD

**Issue by MartinuzziFrancesco** [SciML/FluxNeuralOperators.jl#99]
The links in the readme to Fourier neural operator (https://github.com/SciML/NeuralOperators.jl/blob/main/src/model.jl), Markov neural operator (https://github.com/SciML/NeuralOperators.jl/blob/main/src/model.jl) and DeepONet operator (https://github.com/SciML/NeuralOperators.jl/blob/main/src/DeepONet.jl) are returning 404s

**Comment by ChrisRackauckas** [comment]
All of these should just point to the docs. @ArnoStrouwen has this library gotten the full treatment?

**Comment by ArnoStrouwen** [comment]
Yes, it did. But there are some relative links in this package that might be outdated.
